src/Picker/DcaPickerProviderInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Picker;
+
+/**
+ * Interface for DCA picker providers.
```

Comment:
I would like to have more description here. `Use this if .....` or something. This would help very much.